meta-mender-imx6ul/templates/local.conf.append

```diff
@@ -0,0 +1,95 @@
+
+# Appended fragment from meta-mender-community/meta-mender-nxp/templates
+# Supported machine is:
+#    - imx6ulevk
+#
+
+# Use separate dir for downloads and cache (for speed up successive builds)
+# Replace '~/' with your 'home' directory (Yocto does not expand ~/)
+# DL_DIR = "~/.yocto/warrior/downloads/"
+# SSTATE_DIR = "~/.yocto/warrior/sstate-cache"
```

Comment:
Even though this is valuable information, there are similar comments in the poky template already, so I think this can just be dropped.

Trying to keep the changes to the minimum and to keep the changes Mender related here.